standalone-metastore/packaging/src/docker/conf/metastore-site.xml

```diff
@@ -32,4 +32,15 @@
     <name>metastore.catalog.servlet.auth</name>
     <value>none</value>
   </property>
+
+  <!--
+    <property>
+      <name>fs.s3a.access.key</name>
+      <value>YOUR_AWS_ACCESS_KEY</value>
+    </property>
```

Comment:
`fs.s3a.endpoint` should be added as well?

Reply:
added